src/Microsoft.AspNetCore.Diagnostics/DeveloperExceptionPage/Views/CompilationErrorPage.cshtml

```diff
@@ -34,6 +34,10 @@
                         <div class="titleerror">@fileName</div>
                     }
                 }
+                @if (!string.IsNullOrEmpty(errorDetail.ErrorMessage))
+                {
+                    <details>@errorDetail.ErrorMessage</details>
```

Comment:
http://caniuse.com/#feat=details indicate that about 76% of browsers in use today correctly support the element. Let's just use a `<div>`.